test/Microsoft.Azure.ServiceBus.UnitTests/API/ApiApprovals.ApproveAzureServiceBus.approved.txt

```diff
@@ -730,6 +730,7 @@ namespace Microsoft.Azure.ServiceBus.Management
     public enum NamespaceType
     {
         ServiceBus = 0,
+        Messaging = 1,
```

Comment:
This is actually not true. Problem is that `ServiceBus` is called `Messaging` in service side. Didnt realize when I renamed the enum. The correct thing is to rename `ServiceBus` to `Messaging`. But that is going to break things. So the alternative that I can think of is to manually handle the deserialization and if it is Messaging, then convert it to ServiceBus. :(